src/shared/main/java/com/couchbase/lite/internal/AbstractExecutionService.java

```diff
@@ -73,6 +73,18 @@ private void finishTask() {
 
             if (latch != null) { latch.countDown(); }
         }
+
+        /* For tests to restart the concurrent executor after stopping it. */
```

Comment:
Would love to annotate this with @VisibleForTesting.  If this is part of the pure Java codebase, though, maybe that won't work...

Reply:
@VisibleForTesting seems to work. I think it's from the same package for non-null annotation.